react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,32 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length > 0) {
+    if (args.length > 1) {
+      return Math.min(...args);
+    }
+    if (isArray(args[0])) {
+      return Math.min(...args[0]);
+    }
+    return Math.min(args[0]);
+  }
+  return undefined;
```

Comment:
In all the `if's` you can delte the curly braces if there is only 1 expression inside. Like this
` if (isArray(args[0])) return Math.min(...args[0]);`